pyriemann_qiskit/classification.py

```diff
@@ -159,9 +161,9 @@ def fit(self, X, y):
         if len(self.classes_) != 2:
             raise Exception("Only binary classification \
                              is currently supported.")
-        y = self._map_classes_to_0_1(y)
 
         class1, class0 = self._split_classes(X, y)
+        y = self._map_classes_to_0_1(y)
```

Comment:
This is not a mistake. The `_map_classes_to_0_1` is now taking place after the `_split_classes`.